tools/lib/template_parser.py

```diff
@@ -184,6 +187,15 @@ def looking_at_whitespace() -> bool:
                 s = get_handlebars_tag(text, state.i)
                 tag = "else"
                 kind = "handlebars_else"
+            elif looking_at_handlebars_triple_stache():
+                s = get_handlebars_triple_stache_tag(text, state.i)
+                variable_name = text[state.i + 3 : state.i + len(s) - 3].strip()
+                if not variable_name.endswith("_html"):
+                    raise TemplateParserError(
+                        "Unescaped variables in triple staches {{{ }}} must be suffixed with `_html`"
+                    )
+                tag = variable_name
```

Comment:
I'm on the fence of whether `variable_name` makes the code more readable here vs. just calling it `tag` from the get-go.  I see the rationale of introducing two names here, but I would probably just use `tag` everywhere.  Your call, since I can see both sides here.